dataset_config/platinum_genomes/bigquery.json

```diff
@@ -1,7 +1,12 @@
-// Fields for BigQuery indexer
+// Configuration for BigQuery indexer
 {
   // Name of BigQuery column that represents primary key. See
   // https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#overview
   // for more information.
-  "primary_key": "catalog_id"
+  "primary_key": "catalog_id",
+
+  "table_names": [
```

Comment:
Naming suggestion: `tables_to_index`

Reply:
I think the "_to_index" part is implied. For example, gcs.json will have gcs_paths. I prefer tables_names/gcs_paths over tables_to_index/gcs_paths_to_index. (Though technically should be table_names_to_index/gcs_paths_to_index.)